gmplot/gmplot.py

```diff
@@ -371,11 +375,12 @@ def write_polyline(self, f, path, settings):
         strokeColor = settings.get('color') or settings.get('edge_color')
         strokeOpacity = settings.get('edge_alpha')
         strokeWeight = settings.get('edge_width')
+        precision = settings.get("precision")
 
         f.write('var PolylineCoordinates = [\n')
         for coordinate in path:
-            f.write('new google.maps.LatLng(%f, %f),\n' %
-                    (coordinate[0], coordinate[1]))
+            f.write('new google.maps.LatLng(%.*f, %.*f),\n' %
+                    (precision, coordinate[0], precision, coordinate[1]))
```

Comment:
Let's allow precision to be modified for all functions that work with `LatLng` - can we add this to `write_map()`, `write_point()`, and `write_heatmap()` as well?

It might be worth creating a function to write `LatLng` with the proper precision (instead of copy-pasting this change across the 5 functions that would benefit from it). Here's what it could look like:
```python
def _format_LatLng(lat, lng, precision):
    return 'new google.maps.LatLng(%.*f, %.*f)' % (precision, lat, precision, lng)
```

Then we would use this in any of the required functions as:
```python
f.write('%s,\n' % (_format_LatLng(coordinate[0], coordinate[1], settings.get("precision")))
```

Reply:
Accept. Although, `write_map()` function doesn't really need that, and passing `settings` into it would cause even more complication. Also there are `CIRCLE`, `X` and `PLUS`, which are handled differently, so this approach doesn't apply as is.